website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -286,7 +286,7 @@ To peer Kubernetes clusters running Consul, you need to create a peering token a
     apiVersion: v1
     kind: Service
     metadata:
-      name: frontend-service
+      name: frontend
```

Comment:
I think L280 may also need to be updated to "backend"

Reply:
Thank you!